src/tests/dbus-tests/test_80_filesystem.py

```diff
@@ -309,6 +315,169 @@ def test_mount_auto(self):
         block_fs.Unmount(self.no_options, dbus_interface=self.iface_prefix + '.Filesystem')
         self.assertFalse(os.path.ismount(mnt_path))
 
+    def test_mount_auto_configurable_mount_options(self):
```

Comment:
We should skip these tests in non-CI environment (see fstab test cases for the skip decorator). We are changing system config file and udev rules, so just to be safe we shouldn't do this when not running in a Jenkins VM.

Reply:
Right, last time a leftover in `/etc/fstab` from our tests prevented my system to boot up. Damned `systemd`.

I've added the `udiskstestcase.TestTags.UNSAFE` decorator also to the `test_80_filesystem.UdisksFSTestCase.test_mount_fstab` test.